zerver/lib/export.py

```diff
@@ -1276,10 +1277,14 @@ def create_soft_link(source: Path, in_progress: bool=True) -> None:
     if in_progress:
         new_target = in_progress_link
     else:
-        subprocess.check_call(['rm', '-f', in_progress_link])
+        try:
+            os.remove(in_progress_link)
+        except FileNotFoundError:
+            pass
         new_target = done_link
 
-    subprocess.check_call(["ln", "-nsf", source, new_target])
+    os.symlink(source, new_target + '.tmp')
+    os.rename(new_target + '.tmp', new_target)
```

Comment:
I kinda want to make a library function in `zulip_tools.py` called `force_symlink` or something similar that we can call in these places, again for readability.

Reply:
Done. (“Force” could mean many things, so I called it `overwrite_symlink`.)